src/components/location/location.service.ts

```diff
@@ -205,6 +205,7 @@ export class LocationService {
         `
           MATCH (location {active: true})-[:name {active:true }]->(name:Property {active: true})
           WHERE name.value CONTAINS $filter
```

Comment:
```suggestion
          WHERE name.value CONTAINS ''
```
This may work but is it the best?